src/machinetalk/support/unionread.c

```diff
@@ -39,19 +39,18 @@ bool print_container(pb_istream_t *stream)
     uint64_t taghdr;
     pb_wire_type_t wiretype;
 
-#if 1
     if (!pb_decode_varint(stream, &taghdr)) {
 	printf("Parsing taghdr failed: %s\n", PB_GET_ERROR(stream));
     }
     tag = taghdr >> 3;
     wiretype = taghdr & 0x07;
-#else
+/**********************************************************
```

Comment:
I think it was just @mhaberler's way of showing an alternative method.
He likes \#if 0 as a way of commenting out :smile:

Lets leave it in plain sight, you won't get your warning and anyone browsing will see it.
Don't know if they would think to look and see if anything had been deleted that might be useful